tests/test_helpers.py

```diff
@@ -15,90 +15,158 @@
-from tests.resources import S3_MOCK_ESSENCE_EVENT, S3_MOCK_INVALID_EVENT, S3_MOCK_ESSENCE_EVENT_WITHOUT_MD5
-
-
-class TestHelperFunctions(unittest.TestCase):
-    def test_get_from_event(self):
-        event_dict = json.loads(S3_MOCK_ESSENCE_EVENT)
-        bucket = get_from_event(event_dict, "bucket")
-        self.assertEqual(bucket, "MAM_HighresVideo")
-        event_name = get_from_event(event_dict, "event_name")
-        self.assertEqual(event_name, "ObjectCreated:Put")
-        or_id = get_from_event(event_dict, "tenant")
-        self.assertEqual(or_id, "OR-rf5kf25")
-
-    def test_normalize_or_id_valid(self):
-        # Arrange
-        or_id = 'or-a1b2c3d'
-        # Act
+
+# External imports
+from pytest import raises
+
+# Internal imports
+from meemoo.helpers import (
+    try_to_find_md5,
+    get_from_event,
+    normalize_or_id,
+    is_event_valid,
+    InvalidEventException,
+)
+from tests.resources import (
+    S3_MOCK_ESSENCE_EVENT,
+    S3_MOCK_INVALID_EVENT,
+    S3_MOCK_ESSENCE_EVENT_WITHOUT_MD5,
+)
+
+
+def test_try_to_find_md5_valid_x_md5sum_meta():
+    # Arrange
+    metadata = {"x-md5sum-meta": "1234abcd1234abcd1234abcd1234abcd"}
+    # Act
+    md5 = try_to_find_md5(metadata)
+    # Assert
+    assert md5 == metadata["x-md5sum-meta"]
+
+
+def test_try_to_find_md5_valid_md5sum():
+    # Arrange
+    metadata = {"md5sum": "1234abcd1234abcd1234abcd1234abcd"}
+    # Act
+    md5 = try_to_find_md5(metadata)
+    # Assert
+    assert md5 == metadata["md5sum"]
+
+
+def test_try_to_find_md5_valid_x_amz_meta_md5sum():
+    # Arrange
+    metadata = {"x-amz-meta-md5sum": "1234abcd1234abcd1234abcd1234abcd"}
+    # Act
+    md5 = try_to_find_md5(metadata)
+    # Assert
+    assert md5 == metadata["x-amz-meta-md5sum"]
+
+
+def test_get_from_event():
+    event_dict = json.loads(S3_MOCK_ESSENCE_EVENT)
+    bucket = get_from_event(event_dict, "bucket")
+    assert bucket == "MAM_HighresVideo"
+    event_name = get_from_event(event_dict, "event_name")
+    assert event_name == "ObjectCreated:Put"
+    or_id = get_from_event(event_dict, "tenant")
+    assert or_id == "OR-rf5kf25"
+
+
+def test_normalize_or_id_valid():
+    # Arrange
+    or_id = "or-a1b2c3d"
+    # Act
+    normalized_or_id = normalize_or_id(or_id)
+    # Assert
+    assert normalized_or_id == "OR-a1b2c3d"
+
+
+def test_normalize_or_id_valid_random():
+    # Arrange
+    or_id = "OR-7p8tc89"
+    # Act
+    normalized_or_id = normalize_or_id(or_id)
+    # Assert
+    assert normalized_or_id == "OR-7p8tc89"
+
+
+def test_normalize_or_id_valid_diff_prefix():
+    # Arrange
+    or_id = "mm-7p8tc89"
+    # Act
+    normalized_or_id = normalize_or_id(or_id)
+    # Assert
+    assert normalized_or_id == "MM-7p8tc89"
+
+
+def test_normalize_or_id_invalid_noid_length_too_short():
+    # Arrange
+    or_id = "or-a1b2c3"
+    # Act and Assert
+    with raises(ValueError) as excinfo:
         normalized_or_id = normalize_or_id(or_id)
```

Comment:
For example:
```suggestion
        _ = normalize_or_id(or_id)
```

Reply:
I removed the variables completely, so the assignment won't cause any confusion. The underscore variable could be used when destructuring, but this is not the case here.